modules/light-clients/08-wasm/types/gas_register.go

```diff
@@ -63,10 +63,13 @@ const (
 
 // default: 0.15 gas.
 // see https://github.com/CosmWasm/wasmd/pull/898#discussion_r937727200
-var defaultPerByteUncompressCost = wasmvmtypes.UFraction{
-	Numerator:   15,
-	Denominator: 100,
-}
+var (
+	defaultPerByteUncompressCost = wasmvmtypes.UFraction{
+		Numerator:   15,
+		Denominator: 100,
+	}
+	VMGasRegister = NewDefaultWasmGasRegister()
```

Comment:
mega nit: breahthing room :D 

```suggestion
	}
	
	VMGasRegister = NewDefaultWasmGasRegister()
```